libs/types/src/lib/service.ts

```diff
@@ -2,7 +2,7 @@ import { MultiLanguageText } from './language';
 
 export interface Service extends ServiceToBeCreated {
   id: string;
-  catalogId: string;
+  published: boolean;
```

Comment:
hva har denne endringen å gjøre med commit-meldingen `feat: add service pages to service-catalog closes`

Reply:
Ingenting, ble noe rebase-krøll 🥴 skal se om jeg får ryddet opp i det